### PR TITLE
Fix: fix C interface for DeePMD-kit

### LINKED ISSUE #1893: The C interface cannot be used to compile ABACUS with DeePMD-kit.
### Describe the bug

I tested the **C interface** of DeePMD-kit recently. However, I found that the `tensorflow_cc` dynamic library must be linked during ABACUS compilation.

Here is the detailed error log: 
[log.txt](https://github.com/deepmodeling/abacus-develop/files/10728338/log.txt)

### Expected behavior

_No response_

### To Reproduce

_No response_

### Environment

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by YuLiu98**
fix #1893

### CODE REVIEW

**Review by njzjz — APPROVED** [review]

File: docs/advanced/install.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ If the Deep Potential model is employed in Molecule Dynamics calculations, the f
 cmake -B build -DDeePMD_DIR=~/deepmd-kit -DTensorFlow_DIR=~/tensorflow
 ```
 
+> `deepmd_c`/`deepmd_cc` and `tensorflow_cc` libraries would be called according to `DeePMD_DIR` and `TensorFlow_DIR`, which is showed in detail in [this page](https://github.com/deepmodeling/deepmd-kit/blob/master/doc/inference/cxx.md).
```
Comment (caic99) [inline docs/advanced/install.md:47]: Using an official docs page would be better?
```suggestion
> `deepmd_c`/`deepmd_cc` and `tensorflow_cc` libraries would be called according to `DeePMD_DIR` and `TensorFlow_DIR`, which is showed in detail in [this page](https://docs.deepmodeling.com/projects/deepmd/en/master/inference/cxx.html).
```

File: docs/advanced/install.md [hunk 2]
```diff
@@ -266,6 +268,8 @@ make DeePMD_DIR=~/deepmd-kit TensorFlow_DIR=~/tensorflow
 
 directly.
 
+> `deepmd_c`/`deepmd_cc` and `tensorflow_cc` libraries would be called according to `DeePMD_DIR` and `TensorFlow_DIR`, which is showed in detail in [this page](https://github.com/deepmodeling/deepmd-kit/blob/master/doc/inference/cxx.md).
```
Comment (caic99) [inline docs/advanced/install.md:271]:
```suggestion
> `deepmd_c`/`deepmd_cc` and `tensorflow_cc` libraries would be called according to `DeePMD_DIR` and `TensorFlow_DIR`, which is showed in detail in [this page](https://docs.deepmodeling.com/projects/deepmd/en/master/inference/cxx.html).
```

File: cmake/FindTensorFlow.cmake [hunk 3]
```diff
@@ -0,0 +1,29 @@
+###############################################################################
+# - Find TensorFlow
```
Comment (caic99) [inline cmake/FindTensorFlow.cmake:2]: Does TF have its own cmake module file?

**Review by caic99 — APPROVED** [review]

File: docs/advanced/install.md (hunk 1 above)
Comment (njzjz) [reply docs/advanced/install.md:47]: It's better to provide a link to the GitHub Release page to download the prebuilt libraries directly.

File: cmake/FindTensorFlow.cmake (hunk 3 above)
Comment (YuLiu98) [reply cmake/FindTensorFlow.cmake:2]: > Does TF have its own cmake module file?

I didn't find it, maybe we could use this file temporarily.

File: docs/advanced/install.md (hunk 1 above)
Comment (YuLiu98) [reply docs/advanced/install.md:47]: Both ways are OK for me.

**Pull request merged**